packages/web/management/other/index.php

```diff
@@ -123,7 +135,10 @@
     echo '</div>';
     echo '</div>';
 } else {
-    echo '<nav class="navbar navbar-inverse navbar-fixed-top">';
+    //CES_CUSTOMIZATION 20220323 START     
+    //echo '<nav class="navbar navbar-inverse navbar-fixed-top">';
+    //echo '<nav class="navbar navbar-fixed-top" style="background-color: #f8f8f8;border-color: #337AB7">';
```

Comment:
remove

Reply:
removed 140 and remain the ori code 139 on next release